view/src/components/Profile.js

```diff
@@ -74,13 +49,9 @@ class Profile extends React.Component {
   
 
   render() {
-    const profile = this.props.profile;
-    if (!profile) {
-      return null;
-    }
-
+    var teacher = this.state.teacher;
     // TEST ARRAY: THIS WILL BE REPLACED WITH API RESPONSE
-    var available = [
+    var test = [
```

Comment:
should we delete it to cleanup?

Reply:
We should leave it for now, until I haven’t figured out a way to show the actual availability array.

Reply:
oh I thought you figured that out.. okay let's leave it